src/styles/typography.js

```diff
@@ -17,16 +17,16 @@ export default function createTypography() {
     },
   });
   const primary = ({ weight = 'regular' } = {}) => ({
-    ...ifSmallDevice(14, 16),
+    ...ifSmallDevice(16, 16),
```

Comment:
does it make sense to have a function then? I mean it includes media query, so that will be included in all primary texts

Reply:
and perhaps will slow down the paint/draw/layout events

Reply:
Good remarks, fixed